specification/widget/resource-manager/Microsoft.Widget/readme.md

```diff
@@ -1,7 +1,7 @@
-# containerstorage
+# WidgetManagement
```

Comment:
I'll approve the PR now but this readme needs to inside resource-manager folder as per our discussion.